applications/api/biz/handler/api/comment_service.go

```diff
@@ -16,7 +16,15 @@ import (
 )
 
 // CommentAction .
-// @router /douyin/comment/action [POST]
+//
+//	@Tags		Comment评论
```

Comment:
[nit] I think we can use space but tab here.

Reply:
tab is generated when I use command "swag fmt" to format the code, and it automatically use tab. I don't know how to modify it.